spec/services/create_account_spec.rb

```diff
@@ -16,15 +18,38 @@
           ],
         }
       end
+
       let(:expected_result) { ApplicationService::Result.new(true, Account.last, nil) }
 
       it { is_expected.to eql(expected_result) }
     end
 
-    context "when account is not created" do
+    context "when account is created and the user is from fintera" do
       let(:payload) do
         {
-          name: "",
+          name: Faker::Company.name,
```

Comment:
```suggestion
          name: Faker::Company.name,
          entity_name: Faker::Company.name,
```
Aqui faltou o `entity_name`, seria possível reaproveitar este let pros casos em que a conta é criada?